package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@eqworks/kepler.gl",
   "author": "Shan He <shan@uber.com>",
-  "version": "1.1.16",
+  "version": "1.1.16-alpha.1",
```

Comment:
typically when you roll over to alpha, you'd still want to increment the version number (as conceptually `1.1.16-alpha.1 < 1.1.16`)

Reply:
ah, yes, thank you for mentioning that. Amended.